docs/docsite/rst/guide_alicloud.rst

```diff
@@ -0,0 +1,128 @@
+Alibaba Cloud Compute Services Guide
+====================================
+
+.. _alicloud_intro:
+
+Introduction
+````````````
+
+Ansible contains several modules for controlling and managing Alibaba Cloud Compute Services (Alicloud).  The purpose of this
+section is to explain how to put Ansible modules together to use Ansible in Alicloud context.
+
+Requirements for the Alicloud modules are minimal.
+
+All of the modules require recent versions of footmark.  It can be installed from your OS distribution or python's "sudo pip install footmark" in your control machine.
+
+Classically, ansible will execute tasks in remote machines which defined in its hosts, most cloud-control steps occur on your local machine with reference to the regions to control.
+
+Normally, we'll use the following pattern for provisioning steps::
+
+    - hosts: localhost
+      connection: local
+      vars:
+        - ...
+      tasks:
+        - ...
+
+.. _alicloud_authentication:
+
+Authentication
+``````````````
+   
+Authentication with the ALicloud-related modules is handled by either
+specifying your access and secret key as ENV variables or module arguments.
+
+For environment variables::
+
+    export ALICLOUD_ACCESS_KEY='Alicloud123'
+    export ALICLOUD_SECRET_KEY='AlicloudSecret123'
+
+For storing these in a vars_file::
+
+    ---
+    alicloud_access_key: "<your-alicloud_access_key_id>"
+    alicloud_secret_key: "<your-alicloud_access_secret_key>"
+
+Note that if you store your credentials in vars_file, you need to refer to them in each Alicloud-module. For example::
+
+    - alicloud_instance:
+      alicloud_access_key: "{{alicloud_access_key}}"
+      alicloud_secret_key: "{{alicloud_secret_key}}"
+      image_id: "..."
+
+.. _alicloud_provisioning:
+
+Provisioning
+````````````
+
+There are a number of modules to create ECS instance, disk, VPC, VSwitch, Security Group and other resources.
+
+An example of making sure there are only 5 instances tagged 'NewECS' and other resources as Alicloud Module follows.
+
+In the example below, the "count" of instances is set to 5. This means if there are 0 instances already existing, then
```

Comment:
When mentioning option names it would be good to have them formatted as code, rather than just in quotes,

```
... would use ``instance_name`` to create... 
```